pylint/message/message_definition_store.py

```diff
@@ -58,9 +58,7 @@ def register_message(self, message: MessageDefinition) -> None:
     # and the arguments are relatively small we do not run the
     # risk of creating a large memory leak.
     # See discussion in: https://github.com/pylint-dev/pylint/pull/5673
-    @functools.lru_cache(  # pylint: disable=method-cache-max-size-none # noqa: B019
-        maxsize=None
-    )
+    @cache  # pylint: disable=method-cache-max-size-none # noqa: B019
```

Comment:
Should we upgrade the comment above ? Do we have a false negative for cache ? I'm not well versed on the difference between cache and lru_cache.

Reply:
`@cache` == `@lru_cache(maxsize=None)`. So I don't think there's anything else to change here.

The pylint message clarifies:
```
 W1518: 'lru_cache(maxsize=None)' or 'cache' will keep all method args alive ...
```